lib/commands/certificate.js

```diff
@@ -0,0 +1,166 @@
+import _ from 'lodash';
+import { fs, plist } from 'appium-support';
+import x509 from 'x509';
+import UUID from 'uuid-js';
+import path from 'path';
+import B from 'bluebird';
+import { waitForCondition } from 'asyncbox';
+import { openUrl } from 'node-simctl';
+
+
+let extensions = {}, commands = {};
+
+const SELF_MODULE_NAME = 'appium-xcuitest-driver';
+const BASE_MODULE_NAME = 'appium-base-driver';
+const CONFIG_EXTENSION = '.mobileconfig';
+
+
+async function toMobileConfig (cert) {
+  let altNames = [];
+  try {
+    altNames = x509.getAltNames(cert);
+  } catch (err) {
+    throw new Error(`Cannot parse the certificate. Is it valid and base64-encoded? ` +
+                    `Original error: ${err.message}`);
+  }
+  const displayName = _.replace(_.last(altNames), /\*/g, '_');
+  if (_.isEmpty(displayName)) {
+    throw new Error(`Cannot parse alternate names from the certificate. ` +
+                    `Original value: ${JSON.stringify(altNames)}`);
+  }
+  const uuid = () => UUID.create().hex.toUpperCase();
```

Comment:
even though it's a different case, i'm not sure i like `uuid` having the same name as `UUID`?

Reply:
maybe `getUUID()`?

Reply:
ok, fixed